pontoon/base/models.py

```diff
@@ -1169,11 +1169,9 @@ def get_latest_activity(cls, self, extra=None):
         """
         latest_translation = None
 
-        if hasattr(self, 'fetched_latest_translation'):
-            if self.fetched_latest_translation:
-                latest_translation = self.fetched_latest_translation[0].latest_translation
-            else:
-                latest_translation = None
+        if hasattr(self, 'fetched_project_locale'):
```

Comment:
Thanks!

I ended up using this: `if getattr(self, 'fetched_project_locale', None):`